riak-admin tictacaae CLI tests

hmmr opened the pull request:
Integration tests for tictacaae CLI riak admin commands (https://github.com/OpenRiak/riak/pull/13, https://github.com/OpenRiak/riak_kv/pull/31; see this [Feature proposal](https://github.com/orgs/OpenRiak/discussions/10)).

hmmr committed:
add riak_admin_tictacaae_console_tests

hmmr committed:
update riak_admin_tictacaae_console_tests to clique output

aef- commented on tests/riak_admin_tictacaae_console_tests.erl:
```diff
@@ -0,0 +1,395 @@
+        fun(Var, Val) ->
+                Cmd = ff("tictacaae ~s -n ~s", [Var, Node1]),
+                Expect = ff("|~s| *~s *|", [Node1, Val]),
+                check_admin_cmd(Node1, Cmd, Expect),
```
This and several others look like they will return `ok|fail` without any assertion.

hmmr replied:
My bad. Addressed in https://github.com/OpenRiak/riak_test/pull/8/commits/aec2318963ea75528ea4a79ed9d2366be0460468.

Commit:
update riak_admin_tictacaae_console_tests to poolsize commands w/ extra arg

Commit:
riak_admin_tictacaae_console_tests: properly check assertions

Commit:
riak_admin_tictacaae_console_tests: fix Expect string in rebuild-schedule

hmmr commented:
Pending reviews in https://github.com/OpenRiak/poolboy/pull/1, https://github.com/OpenRiak/riak_kv/pull/31 and https://github.com/OpenRiak/riak_core/pull/20, to run this test, your riak needs to have riak_kv, riak_core and poolboy deps checked out from branch "tiot/openriak-3.4/tictacaae-and-nextgenrepl-cli-commands" ("..-3.2/.." for poolboy) and symlinked to _checkouts.

Commit:
tictacaae_console_tests: remove tests of poolsize-changing commands

since those commands (rebuildtreeworkers, aaefoldworkers,
rebuildstoreworkers) have been dropped from riak_kv#31
